libs/exporter/src/vault-export/services/vault-export.service.ts

```diff
@@ -260,14 +260,12 @@ export class VaultExportService implements VaultExportServiceAbstraction {
               .filter((c) => c.deletedDate === null)
               .forEach(async (c) => {
                 const cipher = new Cipher(new CipherData(c));
-                exportPromises.push(
-                  (async () => {
-                    const decCipher = await cipher.decrypt(
-                      await this.cipherService.getKeyForCipherKeyDecryption(cipher)
-                    );
-                    decCiphers.push(decCipher);
-                  })()
-                );
+                exportPromises.push(async () => {
+                  const decCipher = await cipher.decrypt(
+                    await this.cipherService.getKeyForCipherKeyDecryption(cipher)
+                  );
+                  decCiphers.push(decCipher);
+                });
```

Comment:
This change breaks the Organization's export functionality.

The extra () is to make it Immediatly-Invoked, without that the code is not properly awaited and decCiphers will be empty.

Having it like @MGibson1 suggested on https://github.com/bitwarden/clients/pull/6241#discussion_r1337805065 works properly and improves readability

 ```suggestion
                exportPromises.push(
                  this.cipherService
                    .getKeyForCipherKeyDecryption(cipher)
                    .then((key) => cipher.decrypt(key))
                    .then((decCipher) => {
                      decCiphers.push(decCipher);
                    })
                );
```